x-pack/plugins/watcher/public/application/sections/watch_status_page/watch_status_page.tsx

```diff
@@ -152,6 +145,13 @@ export const WatchStatus = ({
       setIsActivated(!isActivated);
     };
 
+    const selectedPanel =
```

Comment:
The original logic made it difficult for me to correlate state with rendered output, so I tried to make things more explicit here.